File: VueApp/gerenciador-projetos/src/domain/atividade/Atividade.js
export default class Atividade {

    constructor(atividadeID = 0, codigoIdentificador = '', descricao = '', observacao = '', numeroOrdenacao = '', valoresEsforco = '', esforcoEstimado = '', esforcoReal = '', projetoID = 0, SprintID = 0, colunaKanbanID = 0) {

        this.atividadeID = atividadeID;
        this.codigoIdentificador = codigoIdentificador;
        this.descricao = descricao;
        this.observacao = observacao;
        this.numeroOrdenacao = numeroOrdenacao;
        this.valoresEsforco = valoresEsforco;
        this.esforcoEstimado = esforcoEstimado;
        this.esforcoReal = esforcoReal;
        this.projetoID = projetoID;
        this.dprintID = dprintID;
        this.colunaKanbanID = colunaKanbanID;
    }

    entidadeDTO(atividade = { atividadeID: 0, codigoIdentificador: '', descricao: '', observacao: '', numeroOrdenacao: '', valoresEsforco: '', esforcoEstimado: '', esforcoReal: '', projetoID: 0, SprintID: 0, colunaKanbanID: 0 }) {

        this.atividadeID = atividade.atividadeID;
        this.codigoIdentificador = atividade.codigoIdentificador;
        this.descricao = atividade.descricao;
        this.observacao = atividade.observacao;
        this.numeroOrdenacao = atividade.numeroOrdenacao;
        this.valoresEsforco = atividade.valoresEsforco;
        this.esforcoEstimado = atividade.esforcoEstimado;
        this.esforcoReal = atividade.esforcoReal;
        this.projetoID = atividade.projetoID;
        this.dprintID = atividade.dprintID;
        this.colunaKanbanID = atividade.colunaKanbanID;
    }
}

File: WebAPI/GerenciadorProjetos/Models/Planejamento/Atividade.cs
using GerenciadorProjetos.Models.Scrum;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace GerenciadorProjetos.Models.Planejamento
{
    [Table("Atividade")]
    public class Atividade
    {
        public int AtividadeID { get; set; }

        [Required]
        public string CodigoIdentificador { get; set; }

        [Required]
        [MaxLength(200), MinLength(3)]
        public string Descricao { get; set; }

        public string Observacao { get; set; }

        public double NumeroOrdenacao { get; set; }

        [MaxLength(500), MinLength(1)]
        public string ValoresEsforco { get; set; }

        public double EsforcoEstimado { get; set; }

        public double EsforcoReal { get; set; }

        public int ProjetoID { get; set; }
        public Projeto Projeto { get; set; }
        
        public int? SprintID { get; set; }
        public Sprint Sprint { get; set; }

        public int? ColunaKanbanID { get; set; }
        public ColunaKanban ColunaKanban { get; set; }

        public ICollection<Tarefa> Tarefas { get; set; }
    }
}